src/openrct2/ride/RideRatings.cpp

```diff
@@ -995,7 +995,7 @@ static void ride_ratings_apply_adjustments(Ride* ride, rating_tuple* ratings)
 
     // Apply total air time
 #ifdef ORIGINAL_RATINGS
-    if (RideData4[ride->type].flags & RIDE_TYPE_FLAG_HAS_AIR_TIME)
+    if (RideTypeDescriptors[ride->type].HasFlag(RIDE_TYPE_FLAG_HAS_AIR_TIME))
```

Comment:
This doesn't really belong to this commit

Reply:
I have to disagree here. This commit removes RideData4 entirely.